teamwork/templates/dashboard.html

```diff
@@ -2,19 +2,30 @@
 {% load extra_tags %}
 
 {% block content %}
+<div id="page-wrapper">
+  <div class="container">
+    <h3>Next Event</h3>
+    
+  </div>
 
-<h1>
-    Dashboard
-</h1>
-<p>"Hello world, and mars"</p>
+  <div class="container">
+    <h3>Ongoing Projects</h3>
+    {% for project in projects %}
+    <div class="jumbotron">
+      <h2><a href="{% url 'project_details' user.username project.name|urlencode_plus %}">{{ project.name }}</a></h2>
+      {% for event in project.event_set.all|dictsort:'date' %}
+      {% if forloop.first %}

```

Comment:
Oh, I see that I forgot to change the html inside `with` tag. I should probably be `{{ events.0.name }} - {{ events.0.date }}`, though I haven't tested it.